src/value.c

```diff
@@ -411,6 +411,10 @@ size_t SIValue_StringConcat(SIValue* strings, unsigned int string_count, char *b
   return offset;
 }
 
+int SIValue_ValuesAreComparable(const SIValue a, const SIValue b) {
```

Comment:
inline?

Reply:
Non-static inline functions are kind of finicky:
https://stackoverflow.com/questions/6312597/is-inline-without-static-or-extern-ever-useful-in-c99

We can make this `static inline` by moving it to `filter_tree.c` or make it a macro - I'd rather not do the definition-in-header proposal seen in that SO link. I think both of those options are perfectly fine, though.